src/main/java/com/coveros/selenified/Selenified.java

```diff
@@ -433,16 +433,16 @@ private void loadInitialPage(App app, String url, OutputFile file) {
             try {
                 app.getDriver().get(url);
                 if (!app.get().url().contains(url)) {
-                    file.recordAction(act, expected,
+                    file.recordStep(act, expected,
                             startingPage + app.get().url() + "</i> loaded instead of <i>" + url + "</i>",
                             Success.FAIL);
                     file.addError();
                     return;
                 }
-                file.recordAction(act, expected, startingPage + url + "</i> loaded successfully", Success.PASS);
+                file.recordStep(act, expected, startingPage + url + "</i> loaded successfully", Success.PASS);

```

Comment:
I guess that one of the things I hate about the html embed is that you can never seem to get the <i> on the same line as the </i>. Even as we migrate to constants, we're always stuck with, "And here's the end of our line formatting."

Drag the formatting into the writer (which is harder than I've implied).